src/test/rpc/AccountSet_test.cpp

```diff
@@ -331,53 +341,82 @@ class AccountSet_test : public beast::unit_test::suite
             auto const amountWithRate =
                 toAmount<STAmount> (multiply(amount.value(), rate));
 
-            env(pay(gw, alice, USD(3)));
-            env(pay(alice, bob, USD(1)), sendmax(USD(3)));
+            env(pay(gw, alice, USD(10)));
+            env.close();
+            env(pay(alice, bob, USD(1)), sendmax(USD(10)));
+            env.close();
 
-            env.require(balance(alice, USD(3) - amountWithRate));
+            env.require(balance(alice, USD(10) - amountWithRate));
             env.require(balance(bob, USD(1)));
         };
 
-        // Test gateway with allowed transfer rates
-        auto const noFix1201 = supported_amendments().reset(fix1201);
-        runTest (Env{*this, noFix1201}, 1.02);
-        runTest (Env{*this, noFix1201}, 1);
-        runTest (Env{*this, noFix1201}, 2);
-        runTest (Env{*this, noFix1201}, 2.1);
-        runTest (Env{*this, supported_amendments()}, 1.02);
-        runTest (Env{*this, supported_amendments()}, 2);
+        // Test gateway with a variety of allowed transfer rates
+        for (double tr = 1.0; tr <= 2.0; tr += 0.03125)
+        {
+            runTest (tr);
```

Comment:
Considering the `runTest` lambda is only called from here, why not just move the code inside the loop?